lightning/src/blinded_path/message.rs

```diff
@@ -366,6 +380,8 @@ impl_writeable_tlv_based_enum!(OffersContext,
 	},
 	(2, InboundPayment) => {
 		(0, payment_hash, required),
+		(1, nonce, required),
+		(2, hmac, required)
```

Comment:
@TheBlueMatt Given `OffersContext::InboundPayment` are used in 0.0.124 for short-lived reply paths, are we ok with breaking serialization? Prior to this PR, they are only used in a reply path for an invoice sent for a refund. So we'd fail parsing a `InvoiceError` responses in that case.